templates/placementapi/config/logging.conf

```diff
@@ -26,6 +26,12 @@ level=WARNING
 formatter=normal
 args=(sys.stdout,)
 
+[handler_fileHandler]
```

Comment:
This looks different to that we do in other places to get the side car logging.
If that intentional?

Reply:
we will resolve that in q1 next year as part of proting the config gen as we will remove this file in that work